### ISSUE TITLE
Feature request: less verbose mode

### ISSUE THREAD

**Issue by xmikos** [AD-Vega/rtl-power-fftw#7]
I have implemented rtl_power_fftw backend in QSpectrumAnalyzer (I will publish it soon), but output to stderr is too much verbose (when there are a lot of hops/repeats). Now I am forwarding stderr to /dev/null, but I would like to still be able to see basic summary info just after rtl_power_fftw starts (only not repeated with each hop/repeat).

Can you please add something like `--quiet` or `--verbosity=<NUM>` option?

**Comment by alajovic** [comment]
I agree - the stderr output of scanning mode is very verbose. Adding a `--quiet` option has been on my mind for a while now, I just have not yet gotten around to implementing it. It should be trivial, though.

**Comment by KlemenBlokar** [comment]
We were already discussing this, yes, but I did not implement it, because librtlsdr is itself writing output to stderr - therefore if we keep quiet, there is still noise from librtlsdr. I was thinking of tackling both issues at once and push patches to librtlsdr, but didn't manage to do it yet.
